### DISCUSSION TITLE
cron.service [UNSAFE]

### DISCUSSION THREAD

**Discussion by LeTink** [CISOfy/lynis#1472] [Q&A] [answered]
Hi,

Could someone point me at some documentation that tells me what this means and how to remediate it if it is indeed a problem?
It doesn't feature in the [doco](https://cisofy.com/documentation/lynis/) as far as I could see, and there's no output pertaining to it in the warnings/suggestions section of the output.

**Comment by mboelen** [comment, marked as answer]
Lynis 3.1.3 will have additional article(s) for suggestions, including the one related to systemd.
